### DISCUSSION TITLE
能否添加一个类似Mybatis SQL语句构建器的功能

### DISCUSSION THREAD

**Discussion by langhaoabcd** [dotnetcore/FreeSql#872] [Ideas]
有时候要动态生成SQL语句，提前并不知道要操作哪张表，Mybatis有SQL语句构建器很方便的使用代码创建Sql语句，Freesql要是有这个功能就好了
[MyBatis SQL语句构建器 ](https://www.w3cschool.cn/mybatis/k8ay1im3.html)

**Comment by luoyunchong** [comment]
see [https://github.com/sqlkata/querybuilder](https://github.com/sqlkata/querybuilder)

**Reply by langhaoabcd** [reply]
@luoyunchong 这个工具好！ 谢谢了
